app/src/main/java/com/xinto/opencord/ui/viewmodel/LoginViewModel.kt

```diff
@@ -7,14 +7,15 @@ import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import com.xinto.opencord.domain.manager.AccountManager
 import com.xinto.opencord.domain.manager.ActivityManager
+import com.xinto.opencord.domain.mapper.toDomain
 import com.xinto.opencord.domain.model.DomainLogin
-import com.xinto.opencord.domain.repository.DiscordAuthRepository
 import com.xinto.opencord.rest.body.LoginBody
 import com.xinto.opencord.rest.body.TwoFactorBody
+import com.xinto.opencord.rest.service.DiscordAuthService
 import kotlinx.coroutines.launch
 
 class LoginViewModel(
-    private val repository: DiscordAuthRepository,
+    private val api: DiscordAuthService,
```

Comment:
same here

Reply:
This is login view model so yes it is needed

Reply:
we could move that to store too but eh later:tm: